opencti-platform/opencti-graphql/src/modules/internal/document/document-domain.ts

```diff
@@ -133,15 +159,14 @@ export const paginatedForPathsWithEnrichment = async (context: AuthContext, user
     orderOptions.orderMode = OrderingMode.Desc;
   }
   const listOptions = { ...opts, ...findOpts, ...orderOptions };
-  const pagination = await listEntitiesPaginated<BasicStoreEntityDocument>(context, user, [ENTITY_TYPE_INTERNAL_FILE], listOptions);
+
+  await checkFileAccess(context, user, entity_id, { entity_id, id: path, filename: '' });
```

Comment:
the third parameter of `checkFileAccess` is `scope` and here it's set with `entity_id`, doesn't seem right, no ?

Reply:
Well, it's the scope for the "UserAction" log, so I think it's a good call to have the entity_id to tell "User A tried to access files of this specific entity and cannot". But I'm open to other thoughts

Reply:
scopes should be a limited list of operations I think, but I don't know well the activity logging part. I can see this graph in a user activity analytics which is a distribution based on scope : 

![image](https://github.com/OpenCTI-Platform/opencti/assets/668192/07d1490c-0043-43c6-9d01-b412b904ea45)

It might not help to have ids in scope

Reply:
You're completely right, my bad I'll change that :)

Reply:
it's even defined in UserActionListener, all `event_scope` are typed with specific values